ui/src/setupTests.js

```diff
@@ -61,6 +62,7 @@ global.afterEach(() => {
 });
 
 enzyme.configure({ adapter: new ReactAdapterWithMountTracking() });
+rtl.configure({ testIdAttribute: 'data-test-id' });
```

Comment:
Configuring RTL to use 'data-test-id' rather than 'data-testid' for its selector:
https://docs.google.com/document/d/1LDxH33OzrWM0-ZpgNrR5WQb8yG5AmaKqd7cZlLTo1v0/edit